core/server/worker/pom.xml

```diff
@@ -109,6 +109,11 @@
       <artifactId>hamcrest</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-testing</artifactId>
```

Comment:
Is this dependency still used after you removed the `GrpcCleanupRule`, etc. ?

Reply:
It's not used. Removed in the new commit.